src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -321,25 +323,115 @@ export default class TaskRoleContainerList extends React.Component {
     }
   }
 
-  getColumns() {
-    const columns = [
+  getColumns(showDebugInfo) {
+    const optionalColumns = [
       {
-        key: 'number',
-        name: 'No.',
+        key: 'nodeName',
+        name: 'Node Name',
         headerClassName: FontClassNames.medium,
-        minWidth: 50,
-        maxWidth: 50,
+        minWidth: 100,
+        maxWidth: 100,
         isResizable: true,
-        onRender: (item, idx) => {
+        onRender: item => {
           return (
-            !isNil(idx) && (
-              <div className={FontClassNames.mediumPlus}>{idx}</div>
-            )
+            <div className={c(FontClassNames.mediumPlus)}>
+              {item.podNodeName}
+            </div>
+          );
+        },
+      },
+      {
+        key: 'exitCode',
+        name: 'Exit Code',
+        headerClassName: FontClassNames.medium,
```

Comment:
By default, also show exitCode, exitType, exitDiagonostic, startTime and completionTime?

Reply:
Prefer to keep exitCode, exitType, exitDiagonostic  as debug mode:
1. If they are default values, the detail list appearance will not that friendly for normal user
2. They actually belong to debug info, so maybe it is reasonable to keep them into debug mode.
3. Maybe startTime and completionTime could be in default mode if needed.

Reply:
User cares about each Task's exit info, they are even more important and precise than stderr and stdout (and any others).

If you run a cmd, won't you ignore the exit code?

Reply:
You can show it very tight, such as:
```
ExitInfo
USER_STOP(-220)
```
And the ```USER_STOP(-220)``` is linked to Show Exit Diag

Reply:
Teams call?